blog/_posts/2017-09-13-Blog-1team-Serv0101.md

```diff
@@ -0,0 +1,38 @@
+---
+title: “Blog- Team Serv0101”
+layout: post
+created_at: Wed Sept 13 2017
+permalink: blog/2017-09-13-final-blog-post-team-Serv0101
+current: blog
+author: Rakhi Sharma And Neha Yadav
+twitter: @atbrakhi
+categories:
+- student-posts
+---
+
+## Blog- Team Serv0101
+
+We have been working on the Servo project, Working on this project has broadened our programming knowledge especially in Rust. We have interacted with Git lot. At this point, we cannot say that we are masters with Rust, but we are much better than when we started learning it.
```

Comment:
Add fullstop at the end of 1st sentence instead of comma. Change lot -> **a** lot, with rust -> **in** Rust